components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/constant/OrganizationManagementConstants.java

```diff
@@ -256,8 +256,12 @@ public enum ErrorMessages {
                 "Organization %s can't manage organization roles."),
         ERROR_CODE_UNAUTHORIZED_ORG_ROLE_ACCESS("60063", "Organization roles can't be managed from a another " +
                 "organization.", "Organization roles of organization %s can't manage from organization %s."),
-        ERROR_CODE_UNAUTHORIZED_ORG_ACCESS("60064", "Organization can only be managed from immediate parent " +
+        ERROR_CODE_UNAUTHORIZED_ORG_ACCESS("60064", "Organization can only be managed from a parent " +
```

Comment:
Since the immediate parent is also an ancestor, and we don't refer the term parent for other ancestors shall we change as follows? WDYT?
```suggestion
        ERROR_CODE_UNAUTHORIZED_ORG_ACCESS("60064", "Organization can only be managed from an ancestor organization " +
```

Reply:
Yes, it is better.  Fixed by - https://github.com/wso2-extensions/identity-organization-management-core/pull/4/commits/7db20455d584144ca94284a12aa7c9581104b471